scripts/kubestellar

```diff
@@ -240,6 +241,24 @@ EOF
     echo "Finished augmenting root:compute for KubeStellar"
 }
 
+function ensure_space_mgt() {
+    kubectl ws root &> /dev/null
+    $SCRIPT_DIR/space-provider-config "$KUBECONFIG" "$SCRIPT_DIR/../config/space/provider_kcp.yaml"
+    if kubectl get Workspace "$space_mgt_name" &> /dev/null; then
+	echo "space-mgt workspace already exists -- using it:"
+	kubectl ws "$space_mgt_name"
+    else
+        kubectl ws create "$space_mgt_name" --enter
+    fi
+    kubectl ws create-context --overwrite
```

Comment:
Sorry for being ignorant - what does this command actually do?

Reply:
.. and why is it needed here?

Reply:
Now I suspect that this context is used in more places.  If it is not just a private detail of one file then it needs to be documented.

Reply:
If it is not just a private detail of one file then it needs to be documented.